### PR TITLE
rehaul form creation component

### PR DESCRIPTION
**Pull request by cmperez**
you know what to do

### CODE REVIEW

File: client/js/components/create_form.jsx [hunk 1]
```diff
@@ -109,144 +109,165 @@ var CreateForm = React.createClass ({
...
+                        <input className="form-input" type="text"
+                               placeholder="input preview" value="" readOnly="true"/>
+                      </div>
+                    );
+                  case "scale":
+                }
+                return (
+                  <div className="slider-preview">
+                    <p className="form-label">{questionTitle}</p>
+                    <output className="slider-output" for={"scale"+questionIndex}
```
Comment (dduugg) [inline client/js/components/create_form.jsx]: super minor: add space around the `+`s

File: client/js/components/create_form.jsx [hunk 2]
```diff
@@ -109,144 +109,165 @@ var CreateForm = React.createClass ({
...
+                          </div>
+                        );
+                      case "scale":
+                        return (
+                          <div className="form-question-block" id={questionActive}
+                            draggable = "true"
+                            onDragStart = {this.dragStart.bind(this, questionIndex)}
+                            onDragEnter = {this.toggleQuestionActive.bind(this, questionIndex)}
+                            onDragLeave = {this.toggleQuestionActive.bind(this, questionIndex)}
+                            onDragOver={this.preventDefault}
```
Comment (dduugg) [inline client/js/components/create_form.jsx]: inconsistent whitespace

File: client/js/components/chart.jsx [hunk 3]
```diff
@@ -97,21 +97,21 @@ var Chart = React.createClass({
...
         bindto: ".visualization-view",
         data: {
             xs: xs,
-            // Data Format: 
+            // Data Format:
             // y row ['dataname', num, num, ...]
             // x row ['dataname_x', num, num, ...]
             columns: transformCSVtoScatterPlot(csvData, schemaName, xIndex, yIndex),
-            type: 'scatter'
+            type: "scatter"
```
Comment (dduugg) [inline client/js/components/chart.jsx]: :clap:

**Comment by dduugg** [comment]
:+1:

**Comment by owen-d** [comment]
Nice formatting

**Comment by cmperez** [comment]
:gun: :gun:

**Pull request merged**